src/server/worldserver/worldserver.conf.dist

```diff
@@ -2897,9 +2897,9 @@ Wintergrasp.NoBattleTimer = 150
 #     Wintergrasp.CrashRestartTimer
 #         Description: Time (in minutes) to delay the restart of Wintergrasp if the world server
 #                      crashed during a running battle.
-#         Default:     10
+#         Default:     15
 
-Wintergrasp.CrashRestartTimer = 10
+Wintergrasp.CrashRestartTimer = 15
```

Comment:
Why are we changing a default value of a conf?

Reply:
Honestly, you can also approve this or not, but it doesn't really matter that much, so i've decided to change this when server crash and battle is on to extend the battle from 10 to 15. :)